src/test/lib/matches_all_pos_list_test.cpp

```diff
@@ -0,0 +1,65 @@
+#include "base_test.hpp"
+#include "gtest/gtest.h"
+#include "operators/get_table.hpp"
+#include "operators/maintenance/create_table.hpp"
+#include "operators/table_wrapper.hpp"
+#include "storage/matches_all_pos_list.hpp"
+#include "storage/pos_list.hpp"
+
+namespace opossum {
+
+class MatchesAllPosListTest : public BaseTest {
+ public:
+  void SetUp() override {
+    column_definitions.emplace_back("a", DataType::Int, false);
+    column_definitions.emplace_back("b", DataType::Float, true);
+
+    dummy_table_wrapper = std::make_shared<TableWrapper>(Table::create_dummy_table(column_definitions));
+    dummy_table_wrapper->execute();
+
+    create_table = std::make_shared<CreateTable>("t", false, dummy_table_wrapper);
+  }
+
+  TableColumnDefinitions column_definitions;
+  std::shared_ptr<TableWrapper> dummy_table_wrapper;
+  std::shared_ptr<CreateTable> create_table;
+};
+
+TEST_F(MatchesAllPosListTest, AddAfterMatchedAllTest) {
+  // This checks if the matchesAllChunk-Flag in the PosList correctly handles rows, that are added to the table
+  // after the posList was created. These later added rows should not be contained in the PosList
+
+  auto table_name = "test_table";
+  auto table = load_table("resources/test_data/tbl/float_int.tbl", 10, false);
```

Comment:
When looking just at this file, it is hard to guess what `false` means. In other places, we uses `enum class X : bool` for this. See, e.g., https://www.bfilipek.com/2017/03/on-toggle-parameters.html

Reply:
I now added ```enum class FinalizeLastChunk : bool { Yes = true, No = false }``` to the ```load_table.hpp```, that should make it clearer